test/unit/mock/files/hosts.allow

```diff
@@ -0,0 +1,17 @@
+#
+# hosts.allow   This file contains access rules which are used to
+#               allow or deny connections to network services that
+#               either use the tcp_wrappers library or that have been
+#               started through a tcp_wrappers-enabled xinetd.
+#
+#               See 'man 5 hosts_options' and 'man 5 hosts_access'
+#               for information on rule syntax.
+#               See 'man tcpd' for information on tcp_wrappers
+#
+# LOCALHOST (ALL TRAFFIC ALLOWED) DO NOT REMOVE FOLLOWING LINE
+ALL: 127.0.0.1, [::1]
+# Allow SSH (you can limit this further using IP addresses - e.g. 192.168.0.*)
+sshd: ALL
+# Added for testing
+LOCAL : [fe80::]/10 : deny
+UNKNOWN, sshd : 127.0.1.154, [:fff:fAb0::] : deny : /etc/bin/
```

Comment:
https://www.centos.org/docs/rhel-rg-en-3/s1-tcpwrappers-access.html

And I see what you're saying about the user experience. maybe it would be better to use the include instead something like:
describe etc_hosts_allow.where (daemon_list include 'sshd') do
    its ('client_list') { should include '192.168.0.1/24' }
end